src/js/main.js

```diff
@@ -1,7 +1,451 @@
+        'Time of Day': 'AM',
+        'Tidal State': 'Low',
+        'Spring/Neap': '',
+        'Wav-File-Name': 'Ish_Bomb20180711_084033_761',
+        'Blast Bearing': 27.05,
+        'Detection Radius(km)': '30'
+      }
+    },
+    {
+      type: 'Feature',
+      geometry: {
+        type: 'Point',
+        coordinates: [39.07604, -5.49252]
+      },
+      properties: {
+        Id: 14,
+        Location: 'Maziwe Island',
+        Date: '7/14/2018',
+        Day: 'Saturday',
+        Time: '12:32:11 PM',
+        'Time of Day': 'PM',
+        'Tidal State': 'Rising',
+        'Spring/Neap': 'Spring',
+        'Wav-File-Name': 'Ish_Bomb20180714_123210_673',
+        'Blast Bearing': 34.84,
+        'Detection Radius(km)': '30'
+      }
+    },
+    {
+      type: 'Feature',
+      geometry: {
+        type: 'Point',
+        coordinates: [39.07604, -5.49252]
+      },
+      properties: {
+        Id: 15,
+        Location: 'Maziwe Island',
+        Date: '7/15/2018',
+        Day: 'Sunday',
+        Time: '8:41:02 AM',
+        'Time of Day': 'AM',
+        'Tidal State': 'Falling',
+        'Spring/Neap': 'Spring',
+        'Wav-File-Name': 'Ish_Bomb20180715_084102_135',
+        'Blast Bearing': 33.23,
+        'Detection Radius(km)': '30'
+      }
+    },
+    {
+      type: 'Feature',
+      geometry: {
+        type: 'Point',
+        coordinates: [39.30479, -4.86934]
+      },
+      properties: {
+        Id: 16,
+        Location: 'Fish Eagle',
+        Date: '8/8/2018',
+        Day: 'Wednesday',
+        Time: '2:29:37 PM',
+        'Time of Day': 'PM',
+        'Tidal State': 'High',
+        'Spring/Neap': '',
+        'Wav-File-Name': 'Ish_Bomb20180808_142936_784',
+        'Blast Bearing': 15.38,
+        'Detection Radius(km)': '30'
+      }
+    },
+    {
+      type: 'Feature',
+      geometry: {
+        type: 'Point',
+        coordinates: [39.30479, -4.86934]
+      },
+      properties: {
+        Id: 17,
+        Location: 'Fish Eagle',
+        Date: '8/21/2018',
+        Day: 'Tuesday',
+        Time: '3:20:55 PM',
+        'Time of Day': 'PM',
+        'Tidal State': 'Falling',
+        'Spring/Neap': '',
+        'Wav-File-Name': 'Ish_Bomb20180821_152054_827',
+        'Blast Bearing': 15.41,
+        'Detection Radius(km)': '30'
+      }
+    },
+    {
+      type: 'Feature',
+      geometry: {
+        type: 'Point',
+        coordinates: [39.07585, -5.49254]
+      },
+      properties: {
+        Id: 18,
+        Location: 'Maziwe Island',
+        Date: '9/29/2018 9:35:14',
+        Day: 'Saturday',
+        Time: '09:35:14AM',
+        'Time of Day': 'AM',
+        'Tidal State': '',
+        'Spring/Neap': '',
+        'Wav-File-Name': 'Ish_Bomb20180929_093514_719',
+        'Blast Bearing': 17.54,
+        'Detection Radius(km)': '30'
+      }
+    }
+  ]
+};
+
+//get geojson coordinates value as center of circle
+let coordinateValues = [];
+Object.keys(blastData.features).forEach(e => {
```

Comment:
// something like this will give you a unique list of coordinates from the GeoJSON object, but this is not correct syntax.

coordinates = {};
blastData.features.forEach( feature
  coordinates[ feature.geometry.coordinates ] = true;
)

Reply:
Decided to use sets and massaged the precision